spec/jobs/scrape_post_job_spec.rb

```diff
@@ -30,7 +30,8 @@
     stub_fixture(url, 'scrape_no_replies')
     board = create(:board)
 
-    expect(ScrapePostJob).to receive(:notify_exception).with(an_instance_of(UnrecognizedUsernameError), url, board.id, nil, Post::STATUS_COMPLETE, false, board.creator_id).and_call_original
+    with = [an_instance_of(UnrecognizedUsernameError), url, board.id, nil, Post::STATUS_COMPLETE, false, board.creator_id]
+    expect(ScrapePostJob).to receive(:notify_exception).with(*with).and_call_original
```

Comment:
Can we do this instead?

```rb
expect(ScrapePostJob).to receive(:notify_exception).with(
  an_instance_of(UnrecognizedUsernameError),
  url,
  board.id,
  nil,
  Post::STATUS_COMPLETE,
  false,
  board.creator_id,
).and_call_original
```

It's a bit long, but it "feels" better than splatting an array here, for me

Reply:
@teceler did you have opinions on this suggestion?

Reply:
it's long and I don't really like it but I can do it if you super want it

Reply:
We could put some of the other params on the same line, so long as other linters don't complain about that!

Maybe this?

```rb
expect(ScrapePostJob).to receive(:notify_exception).with(
  an_instance_of(UnrecognizedUsernameError),
  url, board.id, nil, Post::STATUS_COMPLETE, false, board.creator_id,
).and_call_original
```

The splatting of `with` seems really strange, so I'd prefer this.

Reply:
Is that the only instance of that construction or do I need to hunt for others?

Reply:
The only one, as far as I know!